### PR TITLE
[Optimize flamechart][1/4] Add flamechart to ReactProfilerData

### PR DESCRIPTION
**Pull request by taneliang**
Stack PR by [STACK ATTACK](https://github.com/taneliang/stack-attack):
- #89 Fix lint failure from #88
- **#90 [Optimize flamechart][1/4] Add flamechart to ReactProfilerData**
- #91 [Optimize flamechart][2/4] Replace Speedscope types with custom stack frame type
- #92 [Optimize flamechart][3/4] Rename flame graph -> flame chart
- #93 [Optimize flamechart][4/4] Split FlamechartView into row views

### Summary

This commit begins a stack of PRs that optimizes our flamechart.
Broadly, we need to replace the use of Speedscope's flamechart types
with our own types that are optimized for our rendering code.

As flamechart was always moved around together with ReactProfilerData,
it makes sense to move flamechart into it. That also necessitates moving
preprocessFlamechart into preprocessData.

This PR works towards #50.

### Test Plan

* `yarn flow`: errors present, but no errors in affected code.
* `yarn lint`
* `yarn start`

### CODE REVIEW

File: src/ImportPage.js
Comment (taneliang) [inline src/ImportPage.js:73]: This is from #89's commit, that's also in this PR.

@kartikcho I'll be trying to rebase merge this PR, to see if #89's commit will appear twice on master (let's hope it doesn't). If #89's commit appears twice, I'll force push to the remaining PRs in this stack before merging them, to prevent polluting master.

File: src/util/preprocessData.js
Comment (kartikcho) [inline src/util/preprocessData.js:366]: Nit: Might wanna remove this extra line for no reason.

**Review by kartikcho — APPROVED** [review]
Looks good to me! Wait is that what LGTM means or is it Let's Get This Merged?

File: src/util/preprocessData.js
Comment (taneliang) [reply src/util/preprocessData.js:366]: This empty line was intentional as I want to separate the profiler data declaration that belongs to the rest of this function from the flamechart computation. Imo they're doing different things conceptually so they should be in separate "paragraphs"

File: src/ImportPage.js
Comment (taneliang) [reply src/ImportPage.js:73]: Update: Only one commit appeared on master :D We're all good

**Pull request merged**
